internal/commands/register_buildpack.go

```diff
@@ -0,0 +1,44 @@
+package commands
+
+import (
+	"github.com/spf13/cobra"
+
+	"github.com/buildpacks/pack"
+	"github.com/buildpacks/pack/internal/style"
+
+	"github.com/buildpacks/pack/internal/config"
+	"github.com/buildpacks/pack/logging"
+)
+
+type RegisterBuildpackFlags struct {
+	BuildpackRegistry string
+}
+
+func RegisterBuildpack(logger logging.Logger, cfg config.Config, client PackClient) *cobra.Command {
+	var opts pack.RegisterBuildpackOptions
+	var flags RegisterBuildpackFlags
+
+	cmd := &cobra.Command{
+		Use:   "register-buildpack <image>",
+		Args:  cobra.ExactArgs(1),
+		Short: "Register the buildpack to a registry",
+		RunE: logError(logger, func(cmd *cobra.Command, args []string) error {
+			defaultRegistry, err := cfg.GetRegistry(flags.BuildpackRegistry)
+			if err != nil {
+				return err
+			}
+			opts.ImageName = args[0]
+			opts.Type = defaultRegistry.Type
+			opts.URL = defaultRegistry.URL
+
+			if err := client.RegisterBuildpack(cmd.Context(), opts); err != nil {
+				return err
+			}
+			logger.Infof("Successfully registered %s", style.Symbol(opts.ImageName))
+			return nil
+		}),
+	}
+	cmd.Flags().StringVarP(&flags.BuildpackRegistry, "buildpack-registry", "R", "", "Buildpack Registry name")
```

Comment:
@dfreilich I'm in support of following conventions, so happy to make the change if that is the case. :+1:

Reply:
@dfreilich  I don't think that's the ongoing rule. At least during the initial registry PR I proposed capital `R` because:

> A --buildpack-registry feels like it's of the same importance as --builder.

https://github.com/buildpacks/pack/pull/540#pullrequestreview-401105177

Reply:
I understand the importance, but as a cli user, I would find it annoying to capitalize something (a whole extra button press) which is supposed to make it easier; I find capitalized arguments to be more annoying to remember/use. 

I thought the use of `-B` by `pack build` was more because we needed some way of distinguishing `--buildpack` from `--builder`, rather than to underscore the importance of the argument; and if so, we should follow that pattern in other commands (like `create-builder`, which only has lower case shortened ones)

Reply:
@dfreilich I see your point. at the very minimum I would think we can agree that both `build` and `register-buildpack` should both use the same flag casing.

Reply:
@jromero @dfreilich so what's the take away here?  Do we want to change `-R` to be `-r`?

Reply:
I'd still vote for `-r` over `-R`